persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/StudyMyBatisRepositoryTest.java

```diff
@@ -225,7 +225,6 @@ public void getTags() throws Exception {
 
         CancerStudyTags result = studyMyBatisRepository.getTags("study_tcga_pub");
  
-        Assert.assertEquals((Integer) 1, result.getCancerStudyId());
```

Comment:
@oplantalech why is this test no longer working?

Reply:
@inodb I think I missed something when passing this function through all the layers, and we can retrieve this by using "CancerStudy" instead. Furthermore, this test is about testing if the study tags are retrieved, in any case what I should do is another test to retrieve the cancer study id.

Reply:
testing the studyid in a hardcoded way here is not a good idea anyway, since this id is autoincrement. So I would vote for removing this check. It is not really testing anything useful.